hironx_moveit_config/config/HiroNX.srdf

```diff
@@ -49,14 +49,24 @@
         <link name="LHAND_JOINT2_Link" />
         <link name="LHAND_JOINT3_Link" />
     </group>
+    <group name="both_arm">
+        <group name="right_arm" />
+        <group name="left_arm" />
+    </group>
     <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
-    <group_state name="home" group="left_arm">
```

Comment:
is it ok to remove left arm? do we need init_rtm for left_arm, right_arm and both_arm?